src/templates/home-page/Hero/index.tsx

```diff
@@ -60,10 +60,10 @@ const Hero: FC<{
     `
   )
 
-  const { epoches } = totalStakedData || {
-    epoches: [{ totalStaked: "0" }],
+  const { daometric } = totalStakedData || {
+    daometric: { stakedTotal: "0" },
   }
-  const totalStaked = !error ? epoches[0].totalStaked : "0"
+  const totalStaked = !error ? daometric.stakedTotal : "0"
```

Comment:
The preiovus code works as expected. It looks like we used wrong `!error` here. Should be:

```
  const { epoches } = totalStakedData || {
    epoches: [{ totalStaked: "0" }],
  }
 const totalStaked = !totalStakedError ? epoches[0].totalStaked : "0"   
 ```
 
The `error` comes from second query from `TBTC_SUBGRAPH_ID`.

I would change it back because it returns different staking tvl value than the new code (and it's higher).

Reply:
Fixed in abdaee93e588216abb14cf0860df7cd57c570c65